test/hummingbot/connector/gateway/amm/test_gateway_cancel.py

```diff
@@ -54,7 +54,7 @@ def setUpClass(cls) -> None:
         cls._client_config_map = ClientConfigAdapter(ClientConfigMap())
         cls._connector: GatewayEVMAMM = GatewayEVMAMM(
             client_config_map=cls._client_config_map,
-            connector_name="uniswap",
+            connector_name="amm",
```

Comment:
We have connector / exchange named uniswap
https://hummingbot.org/exchanges/uniswap/

this changes here not needed